### DISCUSSION TITLE
CSV Importer using import data instead of date in csv file

### DISCUSSION THREAD

**Discussion by pmoberg** [firefly-iii/firefly-iii#4434] [General]
CSV Importer v.2.5.0
Firefly III v. 5.4.6

I am trying to import a CSV file from my bank. A line in the csv file looks like this:

"02/01/2021", "-100.0", "*" ,"" , "DESCRIPTION"

I set up my import configuration with this date format string: `"m-d-Y"`. I assign the role of "Date" to the field in the file but when I import it will always use the import date as the Date. I have also tried just `m-d-Y` for the date format with the same results. 

Any ideas for fix or troubleshooting is appreciated.

Thanks,

**Comment by JC5** [comment]
m/d/Y

**Reply by pmobbe** [reply]
Works good! I realized when I saw your answer how "blind" I had been.
